### ISSUE TITLE
Promo carousel is too fast and not accessible

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-www#100]
The promo carousel animation is distracting, and it's not 
accessible. 

We can added left and right arrows for control. This new version will replace the right to left animation with a fade-in and fade-out effect.

[Accessibility guidance for carousels](https://www.w3.org/WAI/tutorials/carousels/)

### Design

Invision Link: https://projects.invisionapp.com/d/main?origin=v7#/console/18609637/448138461/preview?scrollOffset=1593.3333740234375

### Acceptance criteria

- [ ] no auto-animation
- [ ] when user clicks arrow or dots below the carousel, fade out the current image and fade in the new one
- [ ] make sure the carousel is accessible to screen readers, including alt tags for controls and promo images, and any necessary ARIA attributes

**Comment by noisecapella** [comment]
Is there a mobile mockup for this, or other guidance for how this should look on mobile devices? I assume tablet would look similar to desktop

**Comment by pdpinch** [comment]
It should be [here in InVision](https://projects.invisionapp.com/d/main?origin=v7#/console/18552410/437726301/preview?scrollOffset=0), but I think it needs an update to match the changes to the desktop version.

**Comment by abdulkdawson** [comment]
@noisecapella As far as mobile I think we should add the arrows to indicate the additional slides but remove the animation. Also, we should keep the dotted indicators as well. It should also be swipeable. Hope this helps.
